### PR TITLE
[DOCS-9213] updating dd.trace.span.tags

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
Hotjar feedback
[DOCS-9213](https://datadoghq.atlassian.net/browse/DOCS-9213)

### Merge instructions

DO NOT MERGE.. Waiting on Engineering approval for this setting.

- [ ] Please merge after reviewing

### Additional notes

[DOCS-9213]: https://datadoghq.atlassian.net/browse/DOCS-9213?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rtrieu — APPROVED** [review]
LGTM

**Review by brett0000FF — APPROVED** [review]

File: content/en/tracing/trace_collection/library_config/java.md [hunk 1]
```diff
@@ -179,6 +179,16 @@ A regex to redact sensitive data from incoming requests' query string reported i
...
 
+`dd.trace.span.tags`
+: **Environment Variable**: `DD_TRACE_SPAN_TAGS`<br> 
+**Default**: `null`<br>
+A list of default tags to be added to every span.
+
+`dd.trace.jmx.tags`
+: **Environment Variable**: `DD_TRACE_JMX_TAGS`<br>
+**Default**: `null`<br>
+A list of span tags to be added to every jmx metric.
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply content/en/tracing/trace_collection/library_config/java.md:190]: fixed :)

File: content/en/tracing/trace_collection/library_config/java.md [hunk 2]
```diff
@@ -179,6 +179,16 @@ A regex to redact sensitive data from incoming requests' query string reported i
 **Default**: `true`<br>
 By default, long running asynchronous requests will be marked as an error, setting this value to false allows to mark all timeouts as successful requests.
 
+`dd.trace.span.tags`
+: **Environment Variable**: `DD_TRACE_SPAN_TAGS`<br> 
+**Default**: `null`<br>
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply content/en/tracing/trace_collection/library_config/java.md]: Fixed! @ygree Can you re-review ?

**Pull request merged**